volatility/framework/plugins/windows/vadinfo.py

```diff
@@ -102,17 +108,31 @@ def list_vads(cls, proc: interfaces.objects.ObjectInterface,
 
     @classmethod
     def vad_dump(cls, context: interfaces.context.ContextInterface, proc: interfaces.objects.ObjectInterface,
-                 vad: interfaces.objects.ObjectInterface) -> interfaces.plugins.FileInterface:
+                 vad: interfaces.objects.ObjectInterface, 
+                 maxsize = MAXSIZE_DEFAULT) -> interfaces.plugins.FileInterface:
```

Comment:
I realize this wasn't in the original but should have been, however if we're fixing things up, the signature for this should be `-> Optional[interfaces.plugins.FileInterface]`...

Reply:
I was applying the mods you suggested, and running the tests with vadinfo plugin, I had the same error reported in issue #324.
If you need, I also dumped the values you requested in the original issue, just before the exception:

```
key: address
value: None
```

Reply:
Thanks, could you please report that in bug #324, and make sure that you're testing it with commit `eb02ee1` or later.

Reply:
just done. Tx

Reply:
I'll wait resolution for #324  before committing the mods, otherwise I can not test them.

Reply:
I did a commit including your suggestions.